elide-core/src/main/java/com/yahoo/elide/core/filter/dialect/DefaultFilterDialect.java

```diff
@@ -111,12 +112,19 @@ public FilterExpression parseGlobalExpression(String path, MultivaluedMap<String
                 throw new ParseException(String.format("Invalid predicate: %s", filterPredicate));
             }
 
+            if ((filterPredicate.getOperator().equals(Operator.HASMEMBER)
```

Comment:
This check seems redundant with the validation logic below.  Do we also need this check?

Reply:
This check is for Global Filter. The validation logic below is for the typed filter. Previously we did not have `toMany` relationship check for global filter. So I decided to include only Iscollection check for global filter.

Without this check in global filter, we allow filter expr like `book?filter=title=hasmember='1'` to pass through and jpql rejects these statement.